### PR TITLE
Fix: Increase visibility of services shared by container

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] increases the visibility of services shared by the application container

💁‍♂️ Introducing temporary variables for services fetched from the application container allows to annotate them with the type to which they resolve, and thus increases visibility - making it easier to see which services are actually used, as well as navigating the code.

### CODE REVIEW

File: classes/Http/Controller/Admin/TalksController.php [hunk 1]
```diff
@@ -128,15 +144,21 @@ public function commentCreateAction(Request $req)
...
-        $this->service(TalkHandler::class)
+        /** @var TalkHandler $talkHandler */
+        $talkHandler = $this->service(TalkHandler::class);
+
+        $talkHandler
             ->grabTalk($talkId)
             ->commentOn($req->get('comment'));
 
-        $this->service('session')->set('flash', [
-                'type'  => 'success',
```
Comment (localheinz) [inline classes/Http/Controller/Admin/TalksController.php]: While at it, fixing the indentation here, too.

**Comment by BackEndTea** [comment]
👍 For me if the conflicts get resolved

**Comment by localheinz** [comment]
@BackEndTea 

Rebased!

**Comment by localheinz** [comment]
Thank you, @BackEndTea!

**Pull request merged**